docs/components/tabs.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <title>Control Tabs | Coco &amp; Eve Style Guides</title>
+
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.critical.css">
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.css">
+  <script src="/cocoandeve-styleguides/js/bootstrap.js"></script>
+</head>
+
+<body class="mobile-wrapper">
+  <div class="container mt-4">
+    <h2>Tabs Product</h2>
+  </div>
+  <ul class="nav nav-fill mb-md-7 mb-5 col-md-5 px-0 mx-auto" role="tablist">
+    <li class="nav-item col-4 p-0">
+      <a class="nav-link text-decoration-none" data-toggle="tab" href="#new" role="tab" aria-controls="new">New</a>
+    </li>
+    <li class="nav-item col-4 p-0">
+      <a class="nav-link text-decoration-none active" data-toggle="tab" href="#bestsellers" role="tab" aria-controls="bestsellers">Bestsellers</a>
+    </li>
+    <li class="nav-item col-4 p-0">
+      <a class="nav-link text-decoration-none" data-toggle="tab" href="#value-sets" role="tab" aria-controls="value-sets">Value Sets</a>
+    </li>
+  </ul>
+  <div class="container mt-4">
+    <h1>Tabs Result</h1>
```

Comment:
please review the code. you are applying to much customisations here.
first create a general and global TAB version overwriting Bootstrap vars, which can be used everywhere in the website (check other uses in other pages and create the basic version with all the common properties), second you are not using the Bootstrap class nav-tabs which gives you better management. Honestly I don't see the use of col-4 useful first because -col classes should be inside a .row element, second because with nav-fill you can get the same width of the element.

Reply:
tried with nav-tabs class